src/Resolvers/__tests__/collection_resolvers.test.ts

```diff
@@ -376,4 +446,36 @@ describe("Collection", () => {
       })
     })
   })
+
+  it("returns limited related collections via size arg", () => {
+    const query = `
+      {
+        collection(slug: "jasper-johns-flags") {
+          id
+          title
+          slug
+          category
+          query {
+            id
+            artist_ids
+          }
+          relatedCollections(size: 2) {
+            id
+            slug
+            title
+          }
+        }
+      }
+    `
+
+    return runQuery(query, {}, createMockSchema).then(data => {
+      expect(find).toBeCalledWith({
+        where: {
+          category: { $in: ["Pop Art"] },
+          show_on_editorial: true,
+        },
+        take: 3,
```

Comment:
Similar to an above comment, I think it would be useful to explain *why* we're expecting this call to request 1 more than we passed into the `size` argument.